vlasiator.cpp

```diff
@@ -1173,37 +1069,22 @@ int simulate(int argn,char* args[]) {
 
          phiprof::Timer couplingInTimer {"fsgrid-coupling-in"};
          // Copy moments over into the fsgrid.
-         //setupTechnicalFsGrid(mpiGrid, cells, technicalGrid);
-         feedMomentsIntoFsGrid(mpiGrid, cells, momentsGrid, technicalGrid, false);
-         feedMomentsIntoFsGrid(mpiGrid, cells, momentsDt2Grid, technicalGrid, true);
+         //setupTechnicalFsGrid(mpiGrid, cells, technical, fsgrid);
+         feedMomentsIntoFsGrid(mpiGrid, cells, moments, technical.view(), fsgrid, false);
+         feedMomentsIntoFsGrid(mpiGrid, cells, momentsdt2, technical.view(), fsgrid, true);
          couplingInTimer.stop();
-         
-         propagateFields(
-            perBGrid,
-            perBDt2Grid,
-            EGrid,
-            EDt2Grid,
-            EHallGrid,
-            EGradPeGrid,
-            EGradPeDt2Grid,
-            momentsGrid,
-            momentsDt2Grid,
-            dPerBGrid,
-            dMomentsGrid,
-            dMomentsDt2Grid,
-            BgBGrid,
-            volGrid,
-            technicalGrid,
-            sysBoundaryContainer,
-            P::dt,
-            P::fieldSolverSubcycles
-         );
+
+         propagateFields(perb.view(), perbdt2.view(), e.view(), edt2.view(), ehall.view(), egradpe.view(),
+                         egradpedt2.view(), moments.view(), momentsdt2.view(), dperb.view(), dmoments.view(),
+                         dmomentsdt2.view(), bgb.view(), vol.view(), technical.view(), fsgrid, sysBoundaryContainer,
+                         P::dt, P::fieldSolverSubcycles);
```

Comment:
I don't think that this formatting style makes it more readable. Neither does it when a one-liner is split in two.